### PR TITLE
engine: add release notes for 20.10.21

### PR DESCRIPTION
**Pull request by thaJeztah**
~not yet published, so keeping in draft~

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya @tianon ptal

**Review by tianon — APPROVED** [review]

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+fixes and enhancements.
+
+### Client
+
+- Remove "experimental" gates around "--platform" in bash completion [docker/cli#3824](https://github.com/docker/cli/pull/3824){:target="_blank" rel="noopener"}.
+
+### Daemon
+
+- Allow "allow-nondistributable-artifacts" to be configured for Docker Hub [moby/moby#44313](https://github.com/moby/moby/pull/44313){:target="_blank" rel="noopener"}.
+- Fix an `Invalid standard handle identifie` panic when  registering the docker daemon as a service from a legacy CLI on Windows [moby/moby#44326](https://github.com/moby/moby/pull/44326){:target="_blank" rel="noopener"}.
```
Comment (dvdksn) [inline engine/release-notes/index.md:32]:
```suggestion
- Fix an `Invalid standard handle identifier` panic when registering the Docker daemon as a service from a legacy CLI on Windows [moby/moby#44326](https://github.com/moby/moby/pull/44326){:target="_blank" rel="noopener"}.
```

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+
+### Builder
+
+- Fix running git commands in Cygwin on Windows [moby/moby#44332](https://github.com/moby/moby/pull/44332){:target="_blank" rel="noopener"}.
```
Comment (dvdksn) [inline engine/release-notes/index.md:36]:
```suggestion
- Fix running Git commands in Cygwin on Windows [moby/moby#44332](https://github.com/moby/moby/pull/44332){:target="_blank" rel="noopener"}.
```

File: engine/release-notes/index.md [hunk 3]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+- Update bundled BuildKit version to to fix "output clipped, log limit 1MiB reached" errors [moby/moby#44339](https://github.com/moby/moby/pull/44339){:target="_blank" rel="noopener"}.
```
Comment (dvdksn) [inline engine/release-notes/index.md:37]:
```suggestion
- Update bundled BuildKit version to to fix `output clipped, log limit 1MiB reached` errors [moby/moby#44339](https://github.com/moby/moby/pull/44339){:target="_blank" rel="noopener"}.
```

File: engine/release-notes/index.md [hunk 4]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+
+### Packaging
+
+- Provide packages for Ubuntu 22.10 "Kinetic Kudu".
```
Comment (dvdksn) [inline engine/release-notes/index.md:41]:
```suggestion
- Provide packages for Ubuntu 22.10 Kinetic Kudu.
```

File: engine/release-notes/index.md [hunk 5]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+## 20.10.21
+2022-10-25
+
+This release of Docker Engine contains updated versions of Docker Compose,
+Docker Scan, Containerd, added packages for Ubuntu 22.10, and some minor bug
+fixes and enhancements.
+
+### Client
+
+- Remove "experimental" gates around "--platform" in bash completion [docker/cli#3824](https://github.com/docker/cli/pull/3824){:target="_blank" rel="noopener"}.
```
Comment (dvdksn) [inline engine/release-notes/index.md:27]:
```suggestion
- Remove experimental gate for `--platform` in bash completion [docker/cli#3824](https://github.com/docker/cli/pull/3824){:target="_blank" rel="noopener"}.
```

File: engine/release-notes/index.md [hunk 6]
```diff
@@ -15,6 +15,34 @@ for Docker Engine.
...
+
+### Daemon
+
+- Allow "allow-nondistributable-artifacts" to be configured for Docker Hub [moby/moby#44313](https://github.com/moby/moby/pull/44313){:target="_blank" rel="noopener"}.
```
Comment (dvdksn) [inline engine/release-notes/index.md:31]:
```suggestion
- Add support for `allow-nondistributable-artifacts` towards Docker Hub [moby/moby#44313](https://github.com/moby/moby/pull/44313){:target="_blank" rel="noopener"}.
```

**Review by dvdksn — COMMENTED** [review]
lgtm, some minor suggestions

**Comment by thaJeztah** [comment]
@dvdksn I think I addressed them all; PTAL

**Review by dvdksn — APPROVED** [review]
:shipit:

**Pull request merged**